src/App.js

```diff
@@ -4,7 +4,13 @@ import './App.scss';
 class App extends Component {
```

Comment:
Besides the 2 dev/prod webpack config files, I believe this is the first file you created, yes?